### DISCUSSION TITLE
Accessin my remote repo

### DISCUSSION THREAD

**Discussion by Gabbylim** [community/community#49153] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi everyone?Everytime I try to push or set an  upstream branch  my work to my remote repo  I get an  **kex_exchange_identification: read: Connection reset by peer**.I cant access my account through the ssh.I have been struggling with the issue since last week>no solution has worked.Waiting for a response please
![gitproblem](https://user-images.githubusercontent.com/39884391/222890423-7662ed77-9bc4-450b-a902-6dbc664f69a3.png)

**Comment by airtower-luna** [comment]
Get debug output from SSH using the command `ssh -vT git@github.com`. That will provide a lot more info on what's going on, and hopefully point at a solution.

**Reply by Gabbylim** [reply]
hi?.After running the  command i got this
![image](https://user-images.githubusercontent.com/39884391/222942387-90c0e22d-cf24-4add-926c-7d9ef3eff26b.png)

**Reply by airtower-luna** [reply]
It looks like the connection gets closed before you can even try to authenticate, or the server does. I think it's unlikely GitHub would do that from their end, most likely *something* is blocking the connection. My first guess is a misconfigured firewall.

If you can't solve the problem, you could try using [SSH over the HTTPS port](https://docs.github.com/en/authentication/troubleshooting-ssh/using-ssh-over-the-https-port) to hopefully get around the block.

**Comment by Gabbylim** [comment]
I havent solved yet.This problem came out after accidentally deleting my ssh key...I have tried a new key(rsa) but doesn't seem to work
I have tried resseting,,,,downloading  new open ssh.

**Comment by Gabbylim** [comment]
![image](https://user-images.githubusercontent.com/39884391/223057588-d219184f-ae40-49d4-a5ae-257b0b5ceb4b.png)
I get this..I try using pat or https
